src/server/game/Spells/Spell.h

```diff
@@ -618,7 +618,7 @@ class TC_GAME_API Spell
         SpellMissInfo targetMissInfo;
         SpellEffectHandleMode effectHandleMode;
         // used in effects handlers
-        Unit* unitCaster;
+        Unit* UnitCaster;
```

Comment:
what kind of warning does this trigger? (i dont like the new name - style)

Reply:
Wiki says this is the standard for public/protected members or global variables
https://trinitycore.atlassian.net/wiki/spaces/tc/pages/2130103/C+Development+Standards

Reply:
I think that was written with C++ scripts in mind

Reply:
The cake is a lie

Reply:
jokes aside, I do think its better to follow it